cmd/istio-config-validator/main.go

```diff
@@ -70,6 +69,9 @@ func getFiles(names []string) []string {
 			}
 			return nil
 		})
+		if err != nil {
+			log.Fatal(err.Error())
```

Comment:
Would be nice to change this function to return an error if reasonable.

Reply:
Fully agree, I just did not want to refactor this on this PR. I will add a TODO and follow-up.